### PR TITLE
Remove chain-event Substrate code for related deprecated client code

### PR DESCRIPTION
**Pull request by ianrowan**
## Link to Issue
https://github.com/hicommonwealth/commonwealth/issues/2722

## Description of Changes
- Removes unused substrate logic 
- adds filters to ignore unused event types

## Test Plan
- CW unit tests all pass
- Chain Events unit tests all pass
- Run chain events with `CHAIN=edgeware` and ensured events are flowing properly

### CODE REVIEW

File: packages/chain-events/src/chains/substrate/storageFetcher.ts [hunk 1]
```diff
@@ -141,12 +133,8 @@ export class StorageFetcher extends IStorageFetcher<ApiPromise> {
         return this.fetchDemocracyReferenda(blockNumber, id);
       case EntityKind.SignalingProposal:
         return this.fetchSignalingProposals(blockNumber, id);
-      case EntityKind.TreasuryBounty:
-        return this.fetchBounties(blockNumber, id);
       case EntityKind.TreasuryProposal:
         return this.fetchTreasuryProposals(blockNumber, id);
-      case EntityKind.TipProposal:
```
Comment (jnaviask) [inline packages/chain-events/src/chains/substrate/storageFetcher.ts:148]: Need to keep tips here?

File: packages/chain-events/src/chains/substrate/storageFetcher.ts [hunk 2]
```diff
@@ -211,9 +185,6 @@ export class StorageFetcher extends IStorageFetcher<ApiPromise> {
       ...democracyReferendaEvents,
       ...democracyPreimageEvents,
       ...treasuryProposalEvents,
-      ...bountyEvents,
-      ...technicalCommitteeProposalEvents,
-      ...councilProposalEvents,
```
Comment (jnaviask) [inline packages/chain-events/src/chains/substrate/storageFetcher.ts:216]: We can also remove signaling proposals (dead code even on the blockchain!)

File: packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx [hunk 3]
```diff
@@ -56,7 +56,7 @@ class WhyCommonwealthPage extends ClassComponent {
               project's most important on-chain actions like staking and voting.
               Commonwealth has a growing list of integrations--including
               Ethereum, NEAR, Cosmos, Substrate, and popular DAO frameworks like
-              Moloch or Aragon.
+              or Aragon.
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx:59]: Fix this copy.

**Review by jnaviask — COMMENTED** [review]
Need to make sure we're clear on exactly which Substrate types we're removing. Otherwise, left a few comments. Generally seems good; will re-review after another pass.

File: packages/chain-events/src/chains/substrate/storageFetcher.ts (hunk 2 above)
Comment (ianrowan) [reply packages/chain-events/src/chains/substrate/storageFetcher.ts:216]: Removed in last commit

File: packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx (hunk 3 above)
Comment (ianrowan) [reply packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx]: fixed(i think)

Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx]: nope, just remove the "or" lol

Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx]: "Ethereum, NEAR, Cosmos, Substrate, and popular DAO frameworks
              Aragon"

nope!

Comment (ianrowan) [reply packages/commonwealth/client/scripts/views/pages/why_commonwealth.tsx]: My bad, fixed now 🤦

**Review by jnaviask — APPROVED** [review]

**Pull request merged**